rulesets/best-practices/redocly.yaml

```diff
@@ -0,0 +1,8 @@
+rules:
+  no-server-example.com: warn
+  no-server-trailing-slash: warn
+  no-ambiguous-paths: warn
+  no-path-trailing-slash: warn
+  operation-operationId: warn
+  no-enum-type-mismatch: warn
+  no-unused-components: warn
```

Comment:
Yes I'll apply that ASAP

Reply:
I tried using that "extends" in VSCode with Redocly’s plugin, but I got that error:

Failed to load "https://raw.githubusercontent.com/Redocly/redocly-cli-cookbook/main/rulesets/spec-compliant/redocly.yaml": request to https://raw.githubusercontent.com/Redocly/redocly-cli-cookbook/main/rulesets/spec-compliant/redocly.yaml failed, reason: self signed certificate in certificate chain

Any idea what I did wrong?

Reply:
That's interesting. Let me check it.

Reply:
I wasn't able to reproduce this. Could you file an issue [here](https://github.com/Redocly/redocly-vs-code) with the steps to reproduce the bug and specify your environment (OS, the extension version &c.)?

Reply:
I took this out because we're advising people not to have their tools depend on the repo - nothing here has versions or continuous testing, things can update at any time. The idea is to inspire and have people adopt code and review it for their own use. Instead, I added a link recommending the spec-compliant ruleset to the opening paragraphs.